### PR TITLE
[CIS-1135] Fix user mention suggestions not showing all members

### PR DESCRIPTION
**Pull request by nuno-vieira**
## Description of the pull request
The members that had a missing `lastActiveAt` weren't being fetched. So the sort descriptors should be `lastActiveAt` and the default so that the members and watchers are ordered by last active first, and then the default descriptor, in case the DTO doesn't have `lastActiveAt`

### CODE REVIEW

File: Sources/StreamChatUI/Composer/ComposerVC.swift [hunk 1]
```diff
@@ -646,7 +655,7 @@ open class ComposerVC: _ViewController,
             )
         } else {
             usersCache = searchUsers(
-                channel.watchers.map { $0 } + channel.cachedMembers.map { $0 },
+                channel.watchers.map { $0 } + (memberListController?.members.compactMap { $0 } ?? []),
```
Comment (kojiba) [inline Sources/StreamChatUI/Composer/ComposerVC.swift:658]: `memberListController?.members.compactMap`

won't this impact performance as well? 
iterating on long arrays is N complexity

File: Sources/StreamChatUI/Composer/ComposerVC.swift [hunk 2]
```diff
@@ -244,6 +247,12 @@ open class ComposerVC: _ViewController,
     override open func setUp() {
         super.setUp()
 
+        let client = channelController?.client
+        if let cid = channelController?.cid {
+            memberListController = client?.memberListController(query: .init(cid: cid))
+            memberListController?.synchronize()
```
Comment (kojiba) [inline Sources/StreamChatUI/Composer/ComposerVC.swift:253]: hope this also doing the job not on a main queue

Comment (nuno-vieira) [reply Sources/StreamChatUI/Composer/ComposerVC.swift:253]: Everything on LLC is not doing work on the main thread ofc

File: Sources/StreamChatUI/Composer/ComposerVC.swift (hunk 1 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/Composer/ComposerVC.swift:658]: No, it won't, it's maxed at 100 members. Either way, we should not prematurely optimize for performance, but for readability instead.

**Review by tbarbugli — APPROVED** [review]

**Pull request merged**